CHANGELOG.md

```diff
@@ -6,21 +6,66 @@ accidentally triggering the load of a previous DB version.**
 
 ## 1.3.0 (unreleased)
 
+This release contains major new functionality that we call continuous aggregates.
+
+Aggregate queries which touch large swathes of time-series data can take a long
+time to compute because the system needs to scan large amounts of data on every
+query execution. Our continuous aggregates continuously calculate the
+results of a query in the background and materialize the results. Queries to the
+continuous aggregate view are then significantly faster as they do not need to
+touch the raw data in the hypertable, instead using the pre-computed aggregates
+in the view.
+
+Continuous aggregates are somewhat similar to PostgreSQL materialized
+views, but unlike a materialized view, continuous
+aggregates do not need to be refreshed manually; the view will be refreshed
+automatically in the background as new data is added, or old data is
+modified. Additionally, it does not need to re-calculate all of the data on
+every refresh. Only new and/or invalidated data will be calculated.  Since this
+re-aggregation is automatic, it doesn’t add any maintenance burden to your
+database.
+
+Another alternative approach would be to use triggers to update a materialized view
+immediately upon writing to the database, however this causes significant write
+amplification, which slows inserts. We have designed continuous aggregations to
+run in the background and have a minimal impact on insertion rate by avoiding
+write-amplification.
+
+More information can be found on [our docs overview](http://docs.timescale.com/using-timescaledb/continuous-aggregates)
+or in this [tutorial](http://docs.timescale.com//tutorials/continuous-aggs-tutorial).
+
+**Major Features**
+* #1184 Add continuous aggregate functionality
+
 **Minor Features**
-* #1193 Optimize getting the chunk_id in continuous aggs
-* #1130 Add support for cross datatype chunk exclusion for time types
-* #1112 Add support for window functions to gapfill
-* #1062 Make constraint aware append parallel safe
 * #1005 Enable creating indexes with one transaction per chunk
 * #1007 Remove parent oid from find_children_oids result
 * #1038 Infer time_bucket_gapfill arguments from WHERE clause
+* #1062 Make constraint aware append parallel safe
 * #1067 Add treat_null_as_missing option to locf
+* #1112 Add support for window functions to gapfill
+* #1130 Add support for cross datatype chunk exclusion for time types
 * #1134 Add support for partitionwise aggregation
+* #1139 Add better validation for datatype chunk exclusion
 * #1153 Add time_bucket support to chunk exclusion
+* #1170 Add functions for turning restoring on/off and setting license key
+* #1177 Add transformed time_bucket comparison to quals
+* #1182 Enable optimizing SELECTs within INSERTs
+* #1193 Optimize getting the chunk_id in continuous aggs
```

Comment:
I don't think this should be listed, it's not really a user-facing change, they'll only ever see the optimized version. Same with the others up to and including #1199.